2-projects/1-todo-list/script.js

```diff
@@ -13,13 +42,51 @@ let todos = [
 populateTodoList(todos);
 
 // This function will take the value of the input field and add it as a new todo to the bottom of the todo list. These new todos will need the completed and delete buttons adding like normal.
+
 function addNewTodo(event) {
   // The code below prevents the page from refreshing when we click the 'Add Todo' button.
   event.preventDefault();
   // Write your code here... and remember to reset the input field to be blank after creating a todo!
+
+  const inputBar = document.getElementById("todoInput");
+  const todoText = inputBar.value;
+
+  // If no input then it returns to stop creating todos
+
+  if (!todoText) {
+    return;
+  }
+
+  let newToDo = document.createElement("li");
+  newToDo.classList.add("new-todo-item");
+
+  let todoTextSpan = document.createElement("span");
```

Comment:
It would be good to use common code as the function above for creating an entry so we create it in the same way with complete and remove buttons.

One approach might be to change the data structure of todos and then to call the populate function again! React works in a similar way to this.